pkg/pulumiyaml/run_test.go

```diff
@@ -1975,7 +1991,7 @@ func TestGetConfNodesFromMap(t *testing.T) {
 		t.Run(tt.project, func(t *testing.T) {
 			t.Parallel()
 			result := getConfNodesFromMap(tt.project, tt.propertymap)
-			assert.Equal(t, tt.expected, result)
+			assert.Equal(t, sortConfigNodes(tt.expected), sortConfigNodes(result))
```

Comment:
Might be simpler to use `assert.ElementsMatch` rather than sorting.

Reply:
TIL, that's much better indeed.